### ISSUE TITLE
Issue numbers should be automatically reordered when new issues are added

### ISSUE THREAD

**Issue by haakenlid** [universitas/universitas.no#145]
```
# fix incorrect issue numbers
for iss in Issue.objects.all():
    name = f'{iss.number}/{iss.year}'
    if name != iss.issue_name:
        print(iss.issue_name, name,iss.publication_date, iss.publication_date.weekday())
        iss.save()
        
```

**Comment by siljosaurus** [comment]
We have feedback from the desk that issue numbers should be reordered both when new issues are added or deleted.

**Comment by haakenlid** [comment]
fixed in b53262b
